pyccel/parser/semantic.py

```diff
@@ -1006,10 +1006,17 @@ def _visit_Symbol(self, expr, **settings):
 
     def _visit_DottedVariable(self, expr, **settings):
 
+        if isinstance(expr.lhs, Literal):
```

Comment:
probably , you don't need this anymore. after adding the second error report

Reply:
I still need this as a `Literal` currently doesn't have a `cls_base`

Reply:
can you do this `if hasattr(first, 'cls_base') and  first.cls_base is None:`, with this you can have both cases